x-pack/plugins/infra/server/lib/domains/log_entries_domain/message.ts

```diff
@@ -4,6 +4,8 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import stringify from 'json-stable-stringify';
```

Comment:
We should probably use this in the Flyout code too.

Reply:
makes sense :+1: :arrow_right: #30930

I hope that was what you meant?